src/Runtime/LambdaRuntime.php

```diff
@@ -96,6 +111,13 @@ public function processNextEvent($handler): void
         [$event, $context] = $this->waitNextInvocation();
         \assert($context instanceof Context);
 
+        if ($this->timeout > 0) {
+            Timeout::timoutAfter($this->timeout);
+        } elseif ($this->timeout === 0 && 0 < $context->getRemainingTimeInMillis()) {
+            // Throw exception one second before Lambda pulls the plug.
+            Timeout::timoutAfter((int) floor($context->getRemainingTimeInMillis() / 1000) - 1);
```

Comment:
This could be negative if `$context->getRemainingTimeInMillis()` is between 1 and 999.

Reply:
Thank you. I solved this with `max(1, ...)`